engine/platform/src/platform_window_glfw3.cpp

```diff
@@ -237,6 +247,10 @@ namespace dmPlatform
 
         if (res == PLATFORM_RESULT_OK)
         {
+            // Set size from settings, and show window
+            glfwSetWindowSize(window->m_Window, params.m_Width, params.m_Height);
+            glfwShowWindow(window->m_Window);
```

Comment:
When we apply window size, we show window. 
also calling of `glfwShowWindow` is the fix for the initial issue with invisible window after restart.